api/envoy/config/core/v3/substitution_format_string.proto

```diff
@@ -113,4 +113,7 @@ message SubstitutionFormatString {
   // See the formatters extensions documentation for details.
   // [#extension-category: envoy.formatter]
   repeated TypedExtensionConfig formatters = 6;
+
+  // The output JSON string properties will be sorted if json_format is used and sort_properties is set to true.
+  bool sort_properties = 7;
```

Comment:
is it possible that the end users require new output options in the future? If true, we may need wrapper this to a message like `JsonFormatOptions` or something.

Reply:
Makes sense to me, done